ga4gh/backends.py

```diff
@@ -408,23 +420,27 @@ def searchVariantSets(self, request):
         """
         # This is temporary until we do this properly based on the output
         # buffer size
+        print(request)
         if request.pageSize is None:
             request.pageSize = 100
         response = protocol.GASearchVariantSetsResponse()
-        j = 0
+        currentIndex = 0
         if request.pageToken is not None:
-            j = int(request.pageToken)
-        v = []
-        while j < len(self._variantSetIds) and len(v) < request.pageSize:
-            vs = protocol.GAVariantSet()
-            vs.id = self._variantSetIds[j]
-            vs.datasetId = "NotImplemented"
-            vs.metadata = self._variantSets[vs.id].getMetadata()
-            v.append(vs)
-            j += 1
-        if j < len(self._variantSetIds):
-            response.nextPageToken = j
-        response.variantSets = v
+            currentIndex = int(request.pageToken)
+        variantSetList = []
+        print(self._variantSetIds)
+        while (currentIndex < len(self._variantSetIds) and
+               len(variantSetList) < request.pageSize):
+            variantSet = protocol.GAVariantSet()
+            variantSet.id = self._variantSetIds[currentIndex]
+            variantSet.datasetId = "NotImplemented"
+            variantSet.metadata = self._variantSetMap[variantSet.id]\
```

Comment:
Again, I think we should prefer hanging indents over breaking lines. [PEP 8](https://www.python.org/dev/peps/pep-0008/) says: "The preferred way of wrapping long lines is by using Python's implied line continuation inside parentheses, brackets and braces. Long lines can be broken over multiple lines by wrapping expressions in parentheses. These should be used in preference to using a backslash for line continuation."

Have you run this code through the `pep8` tool?